### DISCUSSION TITLE
About COPY Button and Paste

### DISCUSSION THREAD

**Discussion by AIGC2024** [squidfunk/mkdocs-material#7128] [Q&A]
When copied from my code block and pasted into a text document, there is a blank line between each line of code. 
## Just like this:

` ```yaml `
` line 1 `
` line 2 `
` line 3 `
 ` ``` `

## paste sample:
line2

line2

line3

## The effect of paste I need is：
line1
line2
line3

**Comment by alexvoss** [comment]
This is not generally the case as you can see when you copy and past from the documentation, e.g. [from here](https://squidfunk.github.io/mkdocs-material/plugins/meta/#config.enabled). You could help us investigate the problem by creating a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/).

**Comment by AIGC2024** [comment]
markdown_extensions:
  - pymdownx.highlight:
      #line_spans: __span
Thanks, I have solved this problem， just annotate this line #line_spans: __span
